vision_agent_tools/models/florencev2.py

```diff
@@ -87,36 +88,56 @@ def __call__(
         self,
         task: PromptTask,
         image: Optional[Image.Image] = None,
```

Comment:
can we mark this as deprecating, and we can standardize to use list[Image] moving on (keep it for backward compatibility for now)